docs/feature-gates.md

```diff
@@ -81,3 +82,15 @@ We are currently working on adding documentation for this feature.
 This feature can only be used in "encap" mode when the Geneve tunnel type is
 being used. Note that this is the default configuration for both Linux and
 Windows.
+
+### Flow Exporter
+
+`Flow Exporter` is a feature that runs as part of Antrea Agent, and enables
+to provide network flow visibility into Kubernetes cluster. Flow exporter sends
+IPFIX flow records that are built from observed connections in Conntrack module
+to a flow collector. Refer to this [document](flow-visibility.md) for more information.
+
+#### Requirements for this Feature
+
+This feature is currently only supported for nodes running Linux. In future,
```

Comment:
On the future, support will be extended to Windows.

Reply:
I think you meant "In the"?